tools/ci_build/build.py

```diff
@@ -961,12 +961,14 @@ def generate_build_tree(
         "-Donnxruntime_ENABLE_ROCM_PROFILING=" + ("ON" if args.enable_rocm_profiling else "OFF"),
         "-Donnxruntime_USE_XNNPACK=" + ("ON" if args.use_xnnpack else "OFF"),
         "-Donnxruntime_USE_CANN=" + ("ON" if args.use_cann else "OFF"),
+        "-DMSVC_Z7_OVERRIDE=" + ("ON" if is_windows() and args.use_cache else "OFF"),
```

Comment:
nit: slightly cleaner to only set this within the `if args.use_cache:` below unless there's a reason we need to set the variable for all builds.

Reply:
thx, fixed